ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/CustomerCenterLauncher.kt

```diff
@@ -0,0 +1,53 @@
+package com.revenuecat.purchases.ui.revenuecatui.customercenter
+
+import android.app.Activity
+import android.content.Context
+import android.content.ContextWrapper
+import android.content.Intent
+import com.revenuecat.purchases.ui.revenuecatui.helpers.Logger
+import java.lang.ref.WeakReference
+
+interface CustomerCenterEventHandler {
+    fun onDismissed()
```

Comment:
Just a note that we might want to have default implementations if we add further methods in this interface, otherwise it could be considered a breaking change as well I think. (We should confirm how this would look like for Java though, not sure if this would work well there...)

Reply:
I think they require Android 24? Our minSdk is 21